### PR TITLE
setup dev ows cfg

### PR DESCRIPTION
**Pull request by pindge**
- setup ows cfg for dev instance
- setup crop mask ows layer
- setup ci testing for dev ows cfg

### CODE REVIEW

File: services/ows_refactored/crop_mask/ows_crop_mask_cfg.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    "abstract": "Prob band",
+    "needed_bands": ["prob"],
+    "index_function": {
+        "function": "datacube_ows.band_utils.single_band",
+        "mapped_bands": True,
+        "kwargs": {
+            "band": "prob",
+        },
+    },
+    "color_ramp": [
```
Comment (SpacemanPaul) [inline services/ows_refactored/crop_mask/ows_crop_mask_cfg.py:18]: From what Fang said in the issue, I think we need something like:

```
      "mpl_ramp": "plasma",
      "range": [50,100],
      "legend": {
             "begin": "50",
             "end": "100",
             "ticks_every": "10",
             "tick_labels": {
                   "50": {"prefix": "<"},
             }
      }
```

File: services/ows_refactored/crop_mask/ows_crop_mask_cfg.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    "name": "green",
+    "title": "Green",
+    "abstract": "Classified as water by the decision tree",
+    "value_map": {
+        "crop": [
+            {
+                "title": "Green",
+                "abstract": "Crop",
+                "flags": {"mask": True},
+                "color": "#00ff00",
```
Comment (SpacemanPaul) [inline services/ows_refactored/crop_mask/ows_crop_mask_cfg.py:40]: I would use a darker green than this - maybe #008000

File: services/ows_refactored/crop_mask/ows_crop_mask_cfg.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            },
+            {
+                "title": "No Crop",
+                "abstract": "No crop",
+                "flags": {"mask": False},
+                "color": "#00ff00",
+                "alpha": 0.0
+            },
```
Comment (SpacemanPaul) [inline services/ows_refactored/crop_mask/ows_crop_mask_cfg.py:48]: Also, I would use "Crop" and "No Crop" as the titles, and drop the abstract all together.

Will make for a more readable legend.

File: services/ows_refactored/crop_mask/ows_crop_mask_cfg.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                "flags": {"mask": True},
+                "color": "#ffff00",
+            },
+            {
+                "title": "No Crop",
+                "abstract": "No crop",
+                "flags": {"mask": False},
+                "color": "#ffff00",
+                "alpha": 0.0
+            },
```
Comment (SpacemanPaul) [inline services/ows_refactored/crop_mask/ows_crop_mask_cfg.py:71]: Title: "Crop" or "No Crop".

Remove abstracts.

File: services/ows_refactored/crop_mask/ows_crop_mask_cfg.py [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+style_crop_mask_reversed = {
+    "name": "reversed",
+    "title": "Reversed",
+    "abstract": "Classified as water by the decision tree",
+    "value_map": {
+        "crop": [
+            {
+                "title": "Yellow",
```
Comment (SpacemanPaul) [inline services/ows_refactored/crop_mask/ows_crop_mask_cfg.py:83]: You've reversed the logic, so now the title/abstracts refer to the wrong colours.

**Review by SpacemanPaul — APPROVED** [review]
Good start - Let's see how it goes.

**Review by cbur24 — APPROVED** [review]

**Pull request merged**